src/workerd/io/tracer.c++

```diff
@@ -463,26 +481,37 @@ void BaseTracer::adjustSpanTime(CompleteSpan& span) {
         if (isPredictableModeForTest()) {
           KJ_FAIL_ASSERT("reported span after IoContext was deallocated", span.operationName);
         } else {
-          LOG_WARNING_PERIODICALLY(
-              "reported span after IoContext was deallocated", span.operationName);
+          KJ_LOG(WARNING, "reported span after IoContext was deallocated", span.operationName);
```

Comment:
We didn't encounter the log after landing this, turning to non-periodical so we'll still get a warning if it does happen with a new span.